test/integration/default/serverspec/postgres_spec.rb

```diff
@@ -16,6 +18,6 @@
   it { should be_grouped_into 'postgres' }
 end
 
-describe command('su - postgres -c "psql -qtc \"\l+ db2\""') do
-        its(:stdout) { should match(/db2.*localUser.*UTF8.*C.UTF-8.*C.UTF-8.*my_space/) }
+describe command(%q{su - postgres -c 'psql -qtc "\l+ db2"'}) do
+        its(:stdout) { should match(/db2.*localUser.*UTF8.*en_US\.UTF-8.*en_US\.UTF-8.*my_space/) }
```

Comment:
90cfea1e15006c9781a3c4d2984a0f852b62324c changed the pillar.example data without caring for the spec... 😒